test/performance/odp_dma_perf.c

```diff
@@ -1123,6 +1124,21 @@ static void drain_compl_events(void)
 	}
 }
 
+static void
+drain_poll_transfers(sd_t *sd)
+{
+	const uint32_t count = sd->dma.num_inflight;
+	trs_info_t *infos = sd->dma.infos, *info;
+	odp_dma_t handle = sd->dma.handle;
+	odp_dma_result_t res;
+
+	for (uint32_t i = 0U; i < count; ++i) {
+		info = &infos[i];
```

Comment:
There could be a newline after this assignment and before the `if` below.

Reply:
Ack, I will modify in v2.